sys/syz-extract/linux.go

```diff
@@ -152,6 +159,11 @@ func (*linux) processFile(arch *Arch, info *compiler.ConstInfo) (map[string]uint
 			args = append(args, "-I"+dir)
 		}
 	}
+	if arch.defines != "" {
+		for _, define := range strings.Split(arch.defines, ",") {
```

Comment:
I think `arch` should contain the slice of strings, then you don't need to duplicate `if` and `strings.Split` in every OS.